Blaise-Shyaka/AnnounceIT
# [finishes #170685471] Create responsive home page

# What does this PR do?

This is a welcome/home page for the AnnounceIT application

# Description of tasks to be completed

- Create a home page with signin and signup buttons
- Make it attractive and responsive

# How should this be manually tested? 

- Clone AnnounceIT repository
- cd into /templates/html/index.html and open it
- Adjust your browser window to check its responsiveness
- Check the presence of signin and signup buttons at the top-right corner of the page
